docs/development.adoc

```diff
@@ -158,34 +158,25 @@ $ MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --extra-clusterup-flags "--ena
 
 * `odo` and `oc` binaries in `$PATH`.
 
-.Procedure:
-
-To deploy an integration test:
+.How to write:
 
+Refer odo clean test link:https://github.com/openshift/odo/blob/master/tests/template/template_cleantest_test.go[`template`].
 
-* For the entire integration suite use:
-----
-$ make test-e2e
-----
+.Procedure:
 
-* For the generic tests use:
-----
-$ make test-generic
-----
+Integration tests can be run in the following two ways:
 
-* For the component tests use:
+* To run component test spec in parallel (default) on `4` test node :
++
 ----
 $ make test-cmp-e2e
 ----
++
 
-* For the service catalog tests use:
-----
-$ make test-service-e2e
-----
-
-* For the e2e scenario test use:
+* To run component test spec sequentially or on single test node use enviornment variable `TEST_EXEC_NODES`:
```

Comment:
See other comment. When you say node, you mean an OpenShift node / single master server?

Reply:
> When you say node, you mean an OpenShift node / single master server?

No, here ```node``` means ginkgo provided test node and it has no relation with the number of OpenShift cluster node. 

I have updated the sentence. Does it now make sense.